tools/custom-organization-policy-library/README.md

```diff
@@ -47,12 +55,57 @@ make build
 make constraints                    Build constraint based on input configuration
 make policies                       Build constraint based on input configuration
 make build                          Build constraint and policies based on input configuration
+make deploy-constraint              Build deploy constraint based on input configuration
+make deploy-policy                  Build deploy policy based on input configuration
+make deploy                         Build deploy based on input configuration
+make clean                          Get rid of object and execuatable files
 make config                         Inline Rego rules into constraint templates
 make format                         Format yaml and starlark files
 make help                           Prints help for targets with comments
 make test                           Test custom organization policies
```

Comment:
I know edit as added other stuff in the makefile, to build policies and constaints with Terraform. See with him please